ql/math/randomnumbers/sobolrsg.hpp

```diff
@@ -117,9 +117,10 @@ namespace QuantLib {
         /*! \pre dimensionality must be <= PPMT_MAX_DIM */
         explicit SobolRsg(Size dimensionality,
                           unsigned long seed = 0,
-                          DirectionIntegers directionIntegers = Jaeckel);
+                          DirectionIntegers directionIntegers = Jaeckel,
+                          bool useGrayCode = true);
```

Comment:
Any pointers to docs on using Gray codes?

Reply:
Gray code is used in the current code. I actually added the variant _not_ using the Gray code, since it is needed for the Burley scrambling. This variant is described in the Jaeckel book, formula 8.20. Do you think it's enough to add this reference?

Reply:
Goes to show how much I know :)
Yes, it's probably enough.  Maybe a comment to the effect that one is the "normal" version and the other is used in particular cases?